test/typescript-jest/babel-node/index.test.ts

```diff
@@ -0,0 +1,31 @@
+import fastify from 'fastify'
+import { join } from 'path'
+import AutoLoad from '../../../'
+
+describe("load typescript using babel-node", () => {
+  const app = fastify()
+
+  beforeAll( done => {
+    app.register(AutoLoad, {
+      dir: join(__dirname, '../../commonjs/babel-node/routes')
+    })
+    app.ready(done)
+  })
+
+  it("tests the root route", async function () {
+    const response = await app.inject({
+      method: 'GET',
+      url: '/'
+    })
```

Comment:
if you're GETting root you don't need this variant of inject, simple inject('/')